utils/swift_build_support/swift_build_support/products/swift.py

```diff
@@ -246,6 +249,11 @@ def _enable_experimental_parser_validation(self):
         return [('SWIFT_ENABLE_EXPERIMENTAL_PARSER_VALIDATION:BOOL',
                  self.args.enable_experimental_parser_validation)]
 
+    @property
+    def _swift_debuginfo_non_lto_args(self):
+        return [('SWIFT_DEBUGINFO_NON_LTO_ARGS:STRING',
```

Comment:
I did not handle the case in which the flag is not provided -- in that case we don't want to set this CMake variable, since we want to use either the default value set in CMakeLists.txt or the existing value from CMakeCache.txt

Possibly I would also need to differentiate the default case (`None`?) from setting the flag to an empty value (`[]`?)

Reply:
On further reflection, if the user does not provide the flag at all is best to unset the value from the cache, and let CMake put the default value defined in `CMakeLists.txt` -- this will avoid confusions when iterating at desk on `build-script` invocations.